src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -305,7 +305,13 @@ public function maxSecondsBetweenRedraws(float $seconds): void
     /**
      * Returns an iterator that will automatically update the progress bar when iterated.
      *
-     * @param int|null $max Number of steps to complete the bar (0 if indeterminate), if null it will be inferred from $iterable
+     * @template TKey
+     * @template TValue
+     *
+     * @param iterable<TKey, TValue> $iterable
+     * @param int|null               $max      Number of steps to complete the bar (0 if indeterminate), if null it will be inferred from $iterable
```

Comment:
Could you copy this comment into `SymfonyStyle::progressIterate()`. `null`/`0` meanings needs to be documented.

Reply:
Sure, done.